docs/release-process.md

```diff
@@ -0,0 +1,68 @@
+# The process in more detail
+
+Refer to the [README](../README.md) for a high-level overview of the process.
+
+_NOTE: In these examples the minor version being used is ‘0.42.x’_
+
+## Initialize the process for a new version
+
+1. Create the stable branch with the appropriate name:
+
+   ```bash
+   git checkout master
+   git pull
+   git checkout -b 0.42-stable
+   ```
+
+1. Create a new issue on [the ‘releases’ repo][releases-repo] for the new version and invite users to provide feedback on the RC and what commits [that _have_ landed in `master`] to cherry-pick. (Hereafter referred to as the RC-issue.)
+
+1. Ask the contributors involved in the highlights of this version to start preparing a blog-post to accompany the release.
+
+## Publish the release-candidate
+
+1. Publish the stable branch as an RC. [⇒](./publish-release.md)
+
+1. Create a [GitHub ‘Release’][gh-release] for the new version and include the highlights of this release, so users might be more motivated to give the RC a try and so they can know what to look out for. Include a link to the RC-issue for people to report issues on. (For subsequent RC releases this copy can simply be copied.)
```

Comment:
Should all of this be in the publish release doc instead?

Reply:
I wanted the ‘publish release’ doc to be a doc about the process that leads to creating the release package. This step is part of the larger _conceptual_ release work, but seems somewhat orthogonal to creating the package.

I’m on the fence, though.